### DISCUSSION TITLE
highlight Forestry Leprechaun event Rainbows

### DISCUSSION THREAD

**Discussion by qt31415926535-femboy** [runelite/runelite#17414] [Ideas]
have the Woodcutting plugin also highlight Rainbow tiles or flash an arrow over them
additionally, have the overlay indicate how many stacks of the Rainbow buff you currently have

**Comment by ACarnesi** [comment]
I'd like to see this too. It'd be nice to more easily recognize the rainbow spawns at a glance and it's always struck me as strange that the buff charges weren't displayed in some way. I might take a crack at developing this myself.

**Comment by ScalpelOSRS** [comment]
This would be nice. I Always highlighted them when they spawn so when it spawns on a tile you highlighted earlier, the rainbow will be marked.

**Reply by ACarnesi** [reply]
I've got the changes in a pull request just waiting to be reviewed! Hopefully it gets looked at before too long.

**Comment by ACarnesi** [comment]
Happy to inform this change is live now! Check the woodcutting plugin for the options, but they should be enabled by default. 
![image](https://github.com/user-attachments/assets/aeb524be-8cb4-41ed-8c5d-8edccac38f79)
